.circleci/config.yml

```diff
@@ -27,7 +27,7 @@ jobs:
       - run: |
           bundle install --path=vendor/bundle
 
-      - cache-save:
+      - save_cache:
```

Comment:
I'm sure there's a valid reason but can you explain this change better in the commit message, please?

Reply:
@kennyadsl `cache-save` is not a valid step name in CircleCI, I'm actually not sure how this has worked so far. The right name is `save_cache`.

Reply:
😱

I'm also noticing that we are trying to restore the key `solidus-gems-v1`, which is never saved...

Reply:
@kennyadsl that's actually fine, because it will look for any keys that start with `solidus-gems-v1`.

Reply:
Found some reference about the [`cache-save`](https://thoughtbot.com/blog/circleci-2-rails) step, maybe it's the CircleCI legacy way of calling that type of jobs.